src/components/Wrapper.js

```diff
@@ -3,7 +3,7 @@ import styled, { css } from 'styled-components/native';
 import { normalize } from '../config';
 
 export const Wrapper = styled.View`
-  padding: ${normalize(14)}px;
+  padding: ${normalize(16)}px;
```

Comment:
The padding on this screens is not correct.

|IndexFilter|TilesScreen|
|--|--|
![Simulator Screenshot - iPhone 16 Pro Max - 2024-09-23 at 16 21 04](https://github.com/user-attachments/assets/1ff1dc86-d20a-4e0d-9a93-7424023388a5)|![Simulator Screenshot - iPhone 16 Pro Max - 2024-09-23 at 16 21 55](https://github.com/user-attachments/assets/78de3bd9-d9e9-4ab3-b568-0fd9993053a2)

Reply:
the padding between the tiles is too much.

|before|after|
|--|--|
![IMG_954B25F1DC54-1](https://github.com/user-attachments/assets/74f92c2a-8eb8-4da6-923a-7cf775643b73)|<img width="320" alt="SCR-20240924-mbvs" src="https://github.com/user-attachments/assets/63a5dc91-e645-45a7-8579-10f302dd4a22">

Reply:
ok, fixed

<img width="379" alt="image" src="https://github.com/user-attachments/assets/f48ed8f6-092e-4e5c-bc51-77f96666840a">